Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -329,7 +329,7 @@ final class StatusTableViewController: LoopChartsTableViewController {
         
         // TODO: Don't always assume currentContext.contains(.status)
         reloadGroup.enter()
-        self.deviceManager.loopManager.getLoopState { (manager, state) -> Void in
+        self.deviceManager.loopManager.getLoopState { [weak deviceManager] (manager, state) -> Void in
```

Comment:
Same problem here with `weak deviceManager` as in `Loop/Managers/WatchDataManager.swift`. See that file for long comment.